packages/react-native-editor/ios/Podfile.lock

```diff
@@ -493,12 +493,12 @@ SPEC CHECKSUMS:
   React-logger: faee236598b0f7e1a5e3b68577016ac451f1f993
```

Comment:
These changes were generated. They are unrelated to the implementation details of the component being tested, or the tests.